2015-03-17 20:41:20,664 - [INFO] - from play in pool-6-thread-5 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-03-17 20:41:58,070 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:41:58,259 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:42:01,942 - [INFO] - from application in play-internal-execution-context-1 
Aplicacao inicializada...

2015-03-17 20:42:02,317 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-03-17 20:53:56,898 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:53:56,905 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:53:58,425 - [INFO] - from application in play-internal-execution-context-6 
Aplicacao inicializada...

2015-03-17 20:53:58,431 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-03-17 20:56:13,200 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:56:13,203 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:56:13,970 - [INFO] - from application in play-internal-execution-context-6 
Aplicacao inicializada...

2015-03-17 20:56:13,980 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-03-17 20:57:09,601 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:57:09,605 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:57:09,935 - [INFO] - from application in play-internal-execution-context-6 
Aplicacao inicializada...

2015-03-17 20:57:09,942 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-03-17 20:58:43,063 - [INFO] - from play in play-internal-execution-context-10 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:58:43,066 - [INFO] - from play in play-internal-execution-context-10 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:58:43,260 - [INFO] - from application in play-internal-execution-context-10 
Aplicacao inicializada...

2015-03-17 20:58:43,267 - [INFO] - from play in play-internal-execution-context-10 
Application started (Dev)

2015-03-17 20:59:56,692 - [INFO] - from play in play-internal-execution-context-10 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 20:59:56,695 - [INFO] - from play in play-internal-execution-context-10 
database [default] connected at jdbc:h2:mem:play

2015-03-17 20:59:56,907 - [INFO] - from application in play-internal-execution-context-10 
Aplicacao inicializada...

2015-03-17 20:59:56,922 - [INFO] - from play in play-internal-execution-context-10 
Application started (Dev)

2015-03-17 21:02:11,433 - [INFO] - from play in play-internal-execution-context-12 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:02:11,444 - [INFO] - from play in play-internal-execution-context-12 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:02:11,617 - [INFO] - from application in play-internal-execution-context-12 
Aplicacao inicializada...

2015-03-17 21:02:11,627 - [INFO] - from play in play-internal-execution-context-12 
Application started (Dev)

2015-03-17 21:05:27,605 - [INFO] - from play in play-internal-execution-context-13 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:05:27,608 - [INFO] - from play in play-internal-execution-context-13 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:05:27,770 - [INFO] - from application in play-internal-execution-context-13 
Aplicacao inicializada...

2015-03-17 21:05:27,777 - [INFO] - from play in play-internal-execution-context-13 
Application started (Dev)

2015-03-17 21:06:17,215 - [INFO] - from play in play-internal-execution-context-13 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:06:17,218 - [INFO] - from play in play-internal-execution-context-13 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:06:17,392 - [INFO] - from application in play-internal-execution-context-13 
Aplicacao inicializada...

2015-03-17 21:06:17,397 - [INFO] - from play in play-internal-execution-context-13 
Application started (Dev)

2015-03-17 21:07:53,317 - [INFO] - from play in play-internal-execution-context-13 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:07:53,320 - [INFO] - from play in play-internal-execution-context-13 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:07:53,485 - [INFO] - from application in play-internal-execution-context-13 
Aplicacao inicializada...

2015-03-17 21:07:53,493 - [INFO] - from play in play-internal-execution-context-13 
Application started (Dev)

2015-03-17 21:09:51,459 - [INFO] - from play in play-internal-execution-context-17 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:09:51,461 - [INFO] - from play in play-internal-execution-context-17 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:09:51,628 - [INFO] - from application in play-internal-execution-context-17 
Aplicacao inicializada...

2015-03-17 21:09:51,634 - [INFO] - from play in play-internal-execution-context-17 
Application started (Dev)

2015-03-17 21:10:21,152 - [INFO] - from play in play-internal-execution-context-19 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:10:21,155 - [INFO] - from play in play-internal-execution-context-19 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:10:21,263 - [INFO] - from application in play-internal-execution-context-19 
Aplicacao inicializada...

2015-03-17 21:10:21,274 - [INFO] - from play in play-internal-execution-context-19 
Application started (Dev)

2015-03-17 21:11:03,318 - [INFO] - from play in play-internal-execution-context-19 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:11:03,321 - [INFO] - from play in play-internal-execution-context-19 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:11:03,454 - [INFO] - from application in play-internal-execution-context-19 
Aplicacao inicializada...

2015-03-17 21:11:03,459 - [INFO] - from play in play-internal-execution-context-19 
Application started (Dev)

2015-03-17 21:11:48,669 - [INFO] - from play in play-internal-execution-context-19 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:11:48,672 - [INFO] - from play in play-internal-execution-context-19 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:11:48,802 - [INFO] - from application in play-internal-execution-context-19 
Aplicacao inicializada...

2015-03-17 21:11:48,806 - [INFO] - from play in play-internal-execution-context-19 
Application started (Dev)

2015-03-17 21:13:06,369 - [INFO] - from play in play-internal-execution-context-25 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:13:06,372 - [INFO] - from play in play-internal-execution-context-25 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:13:06,503 - [INFO] - from application in play-internal-execution-context-25 
Aplicacao inicializada...

2015-03-17 21:13:06,507 - [INFO] - from play in play-internal-execution-context-25 
Application started (Dev)

2015-03-17 21:13:44,642 - [INFO] - from play in play-internal-execution-context-27 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:13:44,644 - [INFO] - from play in play-internal-execution-context-27 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:13:44,779 - [INFO] - from application in play-internal-execution-context-27 
Aplicacao inicializada...

2015-03-17 21:13:44,793 - [INFO] - from play in play-internal-execution-context-27 
Application started (Dev)

2015-03-17 21:14:13,264 - [INFO] - from play in play-internal-execution-context-27 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:14:13,267 - [INFO] - from play in play-internal-execution-context-27 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:14:13,399 - [INFO] - from application in play-internal-execution-context-27 
Aplicacao inicializada...

2015-03-17 21:14:13,404 - [INFO] - from play in play-internal-execution-context-27 
Application started (Dev)

2015-03-17 21:16:33,259 - [INFO] - from play in play-internal-execution-context-27 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:16:33,261 - [INFO] - from play in play-internal-execution-context-27 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:16:33,407 - [INFO] - from application in play-internal-execution-context-27 
Aplicacao inicializada...

2015-03-17 21:16:33,413 - [INFO] - from play in play-internal-execution-context-27 
Application started (Dev)

2015-03-17 21:20:53,706 - [INFO] - from play in play-internal-execution-context-31 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:20:53,708 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:20:53,884 - [INFO] - from application in play-internal-execution-context-31 
Aplicacao inicializada...

2015-03-17 21:20:53,887 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2015-03-17 21:22:30,467 - [INFO] - from play in play-internal-execution-context-31 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:22:30,469 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:22:30,645 - [INFO] - from application in play-internal-execution-context-31 
Aplicacao inicializada...

2015-03-17 21:22:30,650 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2015-03-17 21:24:45,409 - [INFO] - from play in play-internal-execution-context-31 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:24:45,413 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:24:45,537 - [INFO] - from application in play-internal-execution-context-31 
Aplicacao inicializada...

2015-03-17 21:24:45,543 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2015-03-17 21:40:27,909 - [INFO] - from play in play-internal-execution-context-31 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 21:40:27,912 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2015-03-17 21:40:28,056 - [INFO] - from application in play-internal-execution-context-31 
Aplicacao inicializada...

2015-03-17 21:40:28,068 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2015-03-17 22:02:00,755 - [ERROR] - from application in New I/O worker #5 


! @6lg5hgf8p - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[error: method render in class index cannot be applied to given types;]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-17 22:03:58,154 - [ERROR] - from application in New I/O worker #5 


! @6lg5hn751 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[error: method render in class index cannot be applied to given types;]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]

2015-03-17 22:04:28,617 - [INFO] - from play in play-internal-execution-context-31 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 22:04:28,620 - [INFO] - from play in play-internal-execution-context-31 
database [default] connected at jdbc:h2:mem:play

2015-03-17 22:04:28,821 - [INFO] - from application in play-internal-execution-context-31 
Aplicacao inicializada...

2015-03-17 22:04:28,826 - [INFO] - from play in play-internal-execution-context-31 
Application started (Dev)

2015-03-17 22:14:11,792 - [ERROR] - from application in New I/O worker #5 


! @6lg5j64o3 - Internal server error, for (GET) [/] ->

play.PlayExceptions$UnexpectedException: Unexpected exception[FileNotFoundException: C:\Users\Andreza\Desktop\projetoSI\ProjetoSI1\target\scala-2.11\classes.bak\sbt7798022450854565539.class (Acesso negado)]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:51) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.1.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.1.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.1.jar:na]
Caused by: java.io.FileNotFoundException: C:\Users\Andreza\Desktop\projetoSI\ProjetoSI1\target\scala-2.11\classes.bak\sbt7798022450854565539.class (Acesso negado)
	at java.io.FileOutputStream.open(Native Method) ~[na:1.8.0_25]
	at java.io.FileOutputStream.<init>(Unknown Source) ~[na:1.8.0_25]
	at java.io.FileOutputStream.<init>(Unknown Source) ~[na:1.8.0_25]
	at sbt.Using$$anonfun$fileOutputChannel$1.apply(Using.scala:82) ~[na:na]
	at sbt.Using$$anonfun$fileOutputChannel$1.apply(Using.scala:82) ~[na:na]
	at sbt.Using$$anon$2.openImpl(Using.scala:72) ~[na:na]
	at sbt.OpenFile$class.open(Using.scala:46) ~[na:na]
	at sbt.Using$$anon$2.open(Using.scala:70) ~[na:na]
	at sbt.Using$$anon$2.open(Using.scala:70) ~[na:na]
	at sbt.Using.apply(Using.scala:24) ~[na:na]
	at sbt.IO$$anonfun$copyFile$3.apply(IO.scala:583) ~[na:na]
	at sbt.IO$$anonfun$copyFile$3.apply(IO.scala:582) ~[na:na]
	at sbt.Using.apply(Using.scala:25) ~[na:na]
	at sbt.IO$.copyFile(IO.scala:582) ~[na:na]
	at sbt.IO$.move(IO.scala:764) ~[na:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2.sbt$inc$ClassfileManager$$anonfun$$anon$$move(ClassfileManager.scala:77) ~[na:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2$$anonfun$delete$3.apply(ClassfileManager.scala:53) ~[na:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2$$anonfun$delete$3.apply(ClassfileManager.scala:52) ~[na:na]
	at scala.collection.immutable.HashSet$HashSet1.foreach(HashSet.scala:153) ~[scala-library-2.11.1.jar:na]
	at scala.collection.immutable.HashSet$HashTrieSet.foreach(HashSet.scala:306) ~[scala-library-2.11.1.jar:na]
	at scala.collection.immutable.HashSet$HashTrieSet.foreach(HashSet.scala:306) ~[scala-library-2.11.1.jar:na]
	at scala.collection.immutable.HashSet$HashTrieSet.foreach(HashSet.scala:306) ~[scala-library-2.11.1.jar:na]
	at sbt.inc.ClassfileManager$$anonfun$transactional$1$$anon$2.delete(ClassfileManager.scala:52) ~[na:na]
	at sbt.inc.Incremental$.prune(Incremental.scala:58) ~[na:na]
	at sbt.inc.IncrementalCommon.cycle(Incremental.scala:96) ~[na:na]
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:38) ~[na:na]
	at sbt.inc.Incremental$$anonfun$1.apply(Incremental.scala:37) ~[na:na]
	at sbt.inc.Incremental$.manageClassfiles(Incremental.scala:65) ~[na:na]
	at sbt.inc.Incremental$.compile(Incremental.scala:37) ~[na:na]
	at sbt.inc.IncrementalCompile$.apply(Compile.scala:27) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile2(AggressiveCompile.scala:157) ~[na:na]
	at sbt.compiler.AggressiveCompile.compile1(AggressiveCompile.scala:71) ~[na:na]
	at sbt.compiler.AggressiveCompile.apply(AggressiveCompile.scala:46) ~[na:na]
	at sbt.Compiler$.apply(Compiler.scala:75) ~[na:na]
	at sbt.Compiler$.apply(Compiler.scala:66) ~[na:na]
	at sbt.Defaults$.sbt$Defaults$$compileTaskImpl(Defaults.scala:770) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:762) ~[na:na]
	at sbt.Defaults$$anonfun$compileTask$1.apply(Defaults.scala:762) ~[na:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library-2.11.1.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_25]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_25]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_25]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_25]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_25]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_25]

2015-03-17 22:18:50,221 - [INFO] - from play in play-internal-execution-context-36 
datasource [jdbc:h2:mem:play;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-03-17 22:18:50,223 - [INFO] - from play in play-internal-execution-context-36 
database [default] connected at jdbc:h2:mem:play

2015-03-17 22:18:50,407 - [INFO] - from application in play-internal-execution-context-36 
Aplicacao inicializada...

2015-03-17 22:18:50,412 - [INFO] - from play in play-internal-execution-context-36 
Application started (Dev)

